comics/comics/ikkesaro.py

```diff
@@ -0,0 +1,18 @@
+# encoding: utf-8
```

Comment:
This line is redundant if there is only ASCII characters in the file, i.e. there is no Norwegian letters.

If I recall correctly, the line actually makes the Python interpreter stop reading the file, change mode, and start over again. So in theory it has negative effect on performance.